### PR TITLE
sdk list use page

### PR DESCRIPTION
**Pull request by alex-smile**
### Description

Fixes # (issue)

### Checklist

- [ ] 填写 PR 描述及相关 issue (write PR description and related issue)
- [ ] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [ ] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

**Review by Han-Ya-Jun — APPROVED** [review]

File: src/dashboard/apigateway/apigateway/apis/web/docs/gateway/gateway/views.py [hunk 1]
```diff
@@ -49,20 +49,20 @@ def list(self, request, *args, **kwargs):
...
                 "gateway_auth_configs": GatewayAuthContext().get_gateway_id_to_auth_config(gateway_ids),
             },
         )
 
-        return self.get_paginated_response(sorted(slz.data, key=lambda x: (-x["is_official"], x["name"])))
+        # 需将官方 SDK 放在前面，但官方标记 is_official 不在 Gateway 表中，因此需要获取数据后，再排序分页
+        page = self.paginate_queryset(sorted(output_slz.data, key=lambda x: (-x["is_official"], x["name"])))
+        return self.get_paginated_response(page)
 
     def _filter_gateways(self, keyword: Optional[str]):
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apis/web/docs/gateway/gateway/views.py:67]: 为啥要提取函数? 看逻辑还得跳, 只有一个viewset用的话直接放进去;

File: src/dashboard/apigateway/apigateway/apis/web/docs/gateway/sdk/views.py [hunk 2]
```diff
@@ -68,7 +65,23 @@ def list(self, request, *args, **kwargs):
                 },
             },
         )
-        return OKJsonResponse(data=output_slz.data)
+        return self.get_paginated_response(output_slz.data)
+
+    def _filter_sdks(self, language: str, keyword: Optional[str]):
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apis/web/docs/gateway/sdk/views.py:70]: 同上, 不需要复用并且逻辑不复杂放回去

Comment (alex-smile) [reply src/dashboard/apigateway/apigateway/apis/web/docs/gateway/sdk/views.py]: done

File: src/dashboard/apigateway/apigateway/apis/web/docs/gateway/gateway/views.py (hunk 1 above)
Comment (alex-smile) [reply src/dashboard/apigateway/apigateway/apis/web/docs/gateway/gateway/views.py]: done

**Review by wklken — APPROVED** [review]

**Pull request merged**